config/routes.rb

```diff
@@ -4,6 +4,9 @@
   post '/login' => 'sessions#create'
   delete '/logout' => 'sessions#destroy'
   get '/signup' => 'users#new'
+  get 'users' => 'admin/organizations#show'
```

Comment:
What is this route for? It seems weird to have `/users` display an organization. Also see my earlier comment regarding replacing the `/admin/organizations/:id` route, I think it should be `admin/organization` instead. Or alternatively, perhaps just `/admin`.

Reply:
I agree. In my opinion, the simplest route `/admin` should display the organization of the logged in user. This would be the landing page after login. In the context of a standard organization member, admin just means administrating my organization.

Later on, when we add the "super admin" functionality, a resource called `/admin/organizations` would be useful as a place to add, edit, and remove organizations from the site.

Reply:
That was how I originally wanted to route it: ```get 'admin'``` but rails kept throwing an error on login saying that "No route matches [GET] '/users'".  So this was really just a workaround for that to accomplish the authentication goal. 

Users don't get routed through the Admin Organizations controller until after they're pointed to admin routes, which happens after login (I'm sure it doesn't have to be that way and there is a way to accomplish this in Routes.rb but I'm not clear on what that should look like specifically).  Maybe the bigger problem is that we currently have 2 Organizations controllers, one is handling the map's data lookups and the other is handling everything else re: organizations.  I'm not sure precisely how to execute this the way we want it, I would need some time to play around with it.  I'm sure it's probably something fairly straightforward that I'm just not seeing.

Reply:
The "No route matches [GET] '/users'" error means that we are trying to redirect to `/users` after login. Changing it to redirect  to `/admin` instead should fix that issue.

The offending line:
https://github.com/my-brothers-keeper/my-brothers-keeper/blob/4887adc3b175690f33678c8214f0a01212449c91/app/controllers/sessions_controller.rb#L8

Reply:
Nice!

Reply:
Wow, that was even more obvious than I suspected.  Thanks!